packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -5077,16 +5079,19 @@ Determines the [IdP Key Credential](#identity-provider-key-credential-object) us
 
 ##### SAML 2.0 Settings object
 
-| Property   | Description                       | DataType    | Nullable | Readonly | DataType                                                             | Default                                               |
-| ---------- | ---------------------             | ----------- | -------- | -------- | -------------------------------------------------------------------- | ----------------------------------------------        |
-| nameFormat | The name identifier format to use | String      | TRUE     | FALSE    | [SAML 2.0 Name Identifier Formats](#saml-2-0-name-identifier-formats) | urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified |
+| Property   | Description                                                                                                  | DataType| Nullable | Readonly | Default                                              |
+| ---------- | ------------------------------------------------------------------------------------------------------------ | ------- | -------- | -------- | ---------------------------------------------------- |
+| nameFormat | The name identifier format to use. See [SAML 2.0 Name Identifier Formats](#saml-2-0-name-identifier-formats) | String  | TRUE     | FALSE    | urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified|
+| honorPersistentNameId | Determines if the IdP should persist account linking when the incoming assertion NameID format is `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent`| Boolean | FALSE | FALSE | FALSE|
```

Comment:
Is Nullable TRUE or FALSE?

Reply:
FALSE right?

Reply:
Just tested, it is Nullable, it is default to FALSE